EIPS/eip-5484.md

```diff
@@ -0,0 +1,101 @@
+  1. [EIP-721](./eip-721.md)’s `ERC721` (`0x80ac58cd`)
+
+- `burnAuth` SHALL be presented to receiver before issuance. 
+- `burnAuth` SHALL be Immutable after issuance.
+- `burnAuth` SHALL be the sole factor that determines which party has the rights to burn token.
+- The issuer SHALL present token metadata to the receiver and acquire receiver's signature before issuance.
+- The issuer SHALL NOT change metadata after issuance.
+
+### Contract Interface
+
+```solidity
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.0;
+
+interface IERC5484 {
+    /// A guideline to standardlize burn-authorization's number coding
+    enum BurnAuth {
+        IssuerOnly,
+        OwnerOnly,
+        Both,
+        Neither
+    }
+
+    /// @notice Emitted when a soulbound token is issued.
+    /// @dev This emit is an add-on to nft's transfer emit in order to distinguish sbt 
+    /// from vanilla nft while providing backward compatibility.
+    /// @param from The issuer
+    /// @param to The receiver
+    /// @param tokenId The id of the issued token
+    event Issued (
+        address indexed from,
+        address indexed to,
+        uint256 indexed tokenId,
+        BurnAuth burnAuth
+    );
+
+    /// @notice provides burn authorization of the token id.
+    /// @dev unassigned tokenIds are invalid, and queries do throw
+    /// @param tokenId The identifier for a token.
+    function burnAuth(uint256 tokenId) external view returns (BurnAuth);
+}
+```
+
+## Rationale
+### SBT as an extension to EIP-721
+We believe that soulbound token serves as a specialized subset of the existing EIP-721 tokens. The advantage of such design is seamless compatibility of soulbound token with existing NFT services. Service providers can treat SBTs like NFTs and do not need to make drastic changes to their existing codebase. 
+
+### Non-Transferable 
+One problem with current soulbound token implementations that extend from [EIP-721](./eip-721.md) is that all transfer implementations throw errors. A much cleaner approach would be for transfer functions to still throw, but also enable thrid parties to check beforehand if the contract implements the soulbound interface to avoid calling transfer. 
+
+### Burn Authorization
+We want maximum freedom when it comes to interface usage. A flexible and predetermined rule to burn is crucial. Here are some sample scenarios for different burn authorizations:
+-  `IssuerOnly`: Loan record
+- `ReceiverOnly`: Paid membership
+- `Both`: Credentials
+- `Neither`: Credit history 
+
+Burn authorization is tied to specific tokens and immutable after issuance. It is therefore important to inform the receiver and gain receiver's consent before the token is issued.
+
+### Issued Event
+On issuing, an `Issued` event will be emitted alongside [EIP-721](./eip-721.md)'s `Transfer` event. This design keeps backward compatibility while giving clear signals to thrid-parties that this is a soulBound token issuance event.
```

Comment:
This just adds a gas cost to every mint with little gain. I would recommend not doing it.